_api-files/embedding-api/objects/sessions/create-a-session.md

```diff
@@ -0,0 +1,37 @@
+---
+content-type: "embed-endpoint"
+endpoint: "sessions"
+key: "create-a-session"
+version: "3"
+order: 1
+
+
+title: "Create a session"
+method: "post"
+short-url: |
+  /v{{ object.version }}{{ object.endpoint-url }}/ephemeral
+full-url: |
+  {{ page.api-base-url }}{{ endpoint.short-url | flatify }}
+description: "Creates a session in the Stitch web application. The session will be for the user for whom the API access token was created."
+
+
+# arguments:
+#   - name: ""
+#     required: 
+#     description: ""
+
+
+returns: "A session object."
+
+examples:
```

Comment:
Do we have a response example for this?

Reply:
Note: After looking through the rest of the doc I noticed that most of the following requests were lacking response examples. I assume you're still working on these - let me know if I can help at all.